src/scripts/routes.js

```diff
@@ -16,52 +16,80 @@ define([
 
     function defineRoute(newRoute) {
         var path = newRoute.path;
-        console.log("Defining route: " + path);
+        console.log("defining route: " + path);
         newRoute.dictionary = "core";
         Emby.Page.addRoute(path, newRoute);
     }
 
-    console.log("Defining core routes");
+    console.log("defining core routes");
 
     defineRoute({
         path: "/addplugin.html",
         autoFocus: false,
         roles: "admin",
-        controller: "addpluginpage"
+        controller: "dashboard/plugins/add"
     });
     defineRoute({
-        path: "/autoorganizelog.html",
-        roles: "admin"
+        path: "/mypreferencesmenu.html",
+        autoFocus: false,
+        transition: "fade",
+        controller: "user/menu"
     });
     defineRoute({
-        path: "/channelsettings.html",
+        path: "/myprofile.html",
         autoFocus: false,
-        roles: "admin"
+        transition: "fade",
+        controller: "user/profile"
     });
     defineRoute({
         path: "/addserver.html",
         autoFocus: false,
         anonymous: true,
         startup: true,
-        controller: "addserver"
+        controller: "auth/addserver"
+    });
+    defineRoute({
+        path: "/mypreferencesdisplay.html",
```

Comment:
why `.html` templates were not moved?

Reply:
None of the templates have been moved so far, only controllers.